changelog/fix18736.dd

```diff
@@ -0,0 +1,23 @@
+Issue 18736 - constructor calls allowed after case labels
+
+Delegating constructor calls are not allowed after labels,
+but case labels and default labels should also count as labels.
+
+  class A
```

Comment:
This looks like MediaWiki syntax? I think you want to add `---` fences to make this a DDoc code block.

Reply:
Yep this is still Ddoc and at the moment it doesn't look so great:

http://dtest.dlang.io/artifact/website-f34af6020cfa432dd56bfd6d65ffcd5e6ed2bce4-8920e4bd03f4a6c8df6bd757a2b9f733/web/changelog/pending.html#fix18736

Reply:
fixed